### PR TITLE
UWA-1848/Endpoint to block a phone

### PR DESCRIPTION
**Pull request by bmoelans**
https://vikingco.atlassian.net/browse/UWA-1848

### CODE REVIEW

File: definitions/telco/block_phone_request.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+type: object
+title: Phone block request
+required:
```
Comment (bmoelans) [inline definitions/telco/block_phone_request.yaml]: Not sure yet the other attributes are optional, asked care but waiting for info

File: paths/telco/block_phone.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+post:
+  tags:
+    - Telco
+  summary: Block phone
+  description: Blocks a phone and suspend related SIM
```
Comment (freeduerinckx) [inline paths/telco/block_phone.yaml]: I don't know if the sim is also suspended

Comment (bmoelans) [reply paths/telco/block_phone.yaml]: Had the same remark myself but forgot to press 'comment'.
It is not done as such by mv-web, but it seems so logic for me.
Block phone has immediate effect (and can be undone)

It is something I will take up in the analysis to discuss with care.

Comment (bmoelans) [reply paths/telco/block_phone.yaml]: Typo, I mean sim block, there are rare scenarios you want to block a phone but not suspend the sim.
Maybe this should be an extra boolean field although you just could do the extra call.

Comment (RoelandMatthijssens) [inline paths/telco/block_phone.yaml]: What do you have against articles? :p `suspends the related SIM`

Comment (freeduerinckx) [reply paths/telco/block_phone.yaml]: While I do agree I would check this with Jens A or Stephanie just to be sure that the requirements are clear.

Comment (jaernouts) [reply paths/telco/block_phone.yaml]: That's actually a very good question. I think how it's working now, is that the Zendesk ticket is just copied to a wholesale ticket, and they handle it further down the line. I don't really know what wholesale does when running that request, I think it's best to check that with Jeroen Moors...

**Comment by bmoelans** [comment]
@RoelandMatthijssens pushed a fixup addressing your remarks

Further @freeduerinckx @jaernouts @Timvde customer care brought to my attention that a phone can have multiple IMEI number, e.g. some dual sim phone not all.

Also discussed some stuff with Jeroen Moors and concluded that it is not a good idea to automatically suspend the sim but it is a good idea to add it as boolean field. Certainly with the extra comment for the case of dual sim phones.

happy to hear your feedback

**Comment by bmoelans** [comment]
@Kjoep and @Akheraphin any remarks from you side on this new end-point to block phone?

**Pull request merged**